projects/plugins/migration/tests/e2e/specs/start.test.js

```diff
@@ -10,6 +10,7 @@ test.describe( 'Starter plugin!', () => {
 		await page.close();
 	} );
 
+	// eslint-disable-next-line playwright/expect-expect -- TODO: Fix/justify this.
```

Comment:
@jeherve Is there some side effect here that we don't need an actual `expect()` call?

Same in a few other files.